docs/features.md

```diff
@@ -714,7 +714,7 @@ Of these four approaches, DoneJS knows about the change the quickest, and update
 To see this in action run the test embedded below that shows how DoneJS, React and Angular compare when updating the DOM when a single property changes:
 <img src="static/img/donejs-minimal-dom-updates-circles.png" alt="Measuring DoneJS, React and Angular rendering a simple property change." />
 
-You can run this test yourself at <a href="http://output.jsbin.com/giyobi/1" target="_blank">JS Bin</a>
+You can run this test yourself at <a href="http://output.jsbin.com/wotevub/" target="_blank">JS Bin</a>
```

Comment:
I _think_ this should be pinned to a specific version (e.g. `wotevub/1`) so that changes aren’t automatically included. I ran into this issue a few months ago: https://github.com/canjs/canjs/pull/2923

Also, might as well link to the `https` versions. :)